create_jenkins_job.py

```diff
@@ -435,6 +436,11 @@ def create_job(os_name, job_name, template_file, additional_dict):
             'tf2_sensor_msgs',
         ]
 
+        foxy_testing_pkgs_for_quality_level = copy.copy(testing_pkgs_for_quality_level)
+        foxy_testing_pkgs_for_quality_level.remove('test_tracetools')
+        foxy_testing_pkgs_for_quality_level.append('tracetools_test')
+        galactic_testing_pkgs_for_quality_level = copy.copy(foxy_testing_pkgs_for_quality_level)
```

Comment:
While this definitely works, it seems a bit fiddly.  I think it would be nicer to me to remove `test_tracetools` from `testing_pkgs_for_quality_level`, then do:

```
ql_testing_pkgs_humble = ['test_tracetools']
ql_testing_pkgs_galactic_foxy = ['tracetools_test']

humble_testing_pkgs = testing_pkgs_for_quality_level + ql_testing_pkgs_humble
galactic_foxy_testing_pkgs = testing_pkgs_for_quality_level + ql_testing_pkgs_galactic_foxy
```

And then use those variables as appropriate below.